manifests/server.pp

```diff
@@ -41,7 +41,7 @@
     ],
   },
 ) {
-  unless $facts['os']['name'] in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
+  if not $facts['os']['name'] in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
```

Comment:
a more readable form, imo, would be
```puppet 
if $facts['os']['name'] not in ['CentOS', 'RedHat', 'VirtuozzoLinux'] {
```
if puppet understands that, that is

Reply:
I guess it's subjective.  To me, the way it currently is (using `unless`), is the most readable.

Reply:
ya, i think we should close this pr